helpers/__tests__/init-test.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Sonar UI Common
+ * Copyright (C) 2019-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+/* eslint-disable no-console */
+import initSonarUICommon, {
+  DEFAULT_L10N_MESSAGES,
+  getL10nLocale,
+  getL10nMessages,
+  getUrlContext,
+  DEFAULT_L10N_LOCALE,
+} from '../init';
+
+const originalConsoleError = console.error;
+console.warn = jest.fn();
+
+beforeEach(() => {
+  (console.warn as jest.Mock).mockClear();
+});
+
+afterAll(() => {
+  initSonarUICommon({ l10nLocale: 'en', l10nMessages: {}, urlContext: '' });
+  console.warn = originalConsoleError;
```

Comment:
You put `console.error` in `console.warn`. Is it on purpose ?

Reply:
OMG no! I switched to warn and forgot one replace. Thank you!